backend/endpoints/train_endpoints.py

```diff
@@ -70,8 +75,35 @@ def tabular_run():
         )
 
         try:
-            createTrainspaceData(tabular_data)
+            create_dynamo_item_from_obj("trainspace", tabular_data)
             print(id)
+
+            async def run():
+                async with aiohttp.ClientSession(json_serialize=json.dumps) as session:
+                    async with session.post(
+                        "http://localhost:8001/api/trainspace/tabular",
```

Comment:
Question:

1. For testing this work locally, would you need to spin up localhost:8001 on a 2nd terminal window along with localhost:8000?
2. How would this implementation work in production. How would our webapp know what localhost:8001 is within AWS docker container? In prod, I imagine that `/backend` would basically call `writeToQueue` and `api/trainspace/tabular` would pretty much ingest the SQS queue entry and train the model. How do we ensure that data flow without being hacky? @dwu359

Reply:
@farisdurrani thoughts?

Reply:
Yes, we would have to spin up both each in their own terminal window in dev, and send it through sqs in prod

Reply:
Actually if there is a way to use sqs in dev we might be able to avoid the aiohttp stuff too

Reply:
idk abt using sqs in dev. unless u spin up a dev queue in sqs, i can see that possibility, but I don't want to risk incurring extra costs on AWS @dwu359

Reply:
Theoretically yes, you would just run `kernel.py` to read from the SQS and run the training according to the training type. It won't work rn since the backend is still undergoing reformat on its data structures but that's in my PR